src/common/scrub_types.cc

```diff
@@ -126,19 +146,24 @@ namespace librados {
 
 void inconsistent_obj_wrapper::encode(bufferlist& bl) const
 {
-  ENCODE_START(1, 1, bl);
+  ENCODE_START(2, 2, bl);
```

Comment:
@athanatos @tchaikov Well, if you upgrade ceph-osd, reboot and scrub and finally use the previous rados command the decode will fail.  We presume that the older version of this feature isn't being used much at this point and the changes I'm making to the results would require rebuilding any other client app.